apps/api/src/types/template.ts

```diff
@@ -0,0 +1,25 @@
+export interface CustomQuestionType {
+  id: number;
+  label: any;
+}
+
+export interface ExerciseTemplate {
+  id?: string;
+  tag: string;
+  title: string;
+  description: string;
+  questions?: number;
+  questionnaire: {
+    questions: {
+      title: string;
+      name: string;
+      points: number;
+      order: number;
+      question_type: CustomQuestionType;
+      options: {
+        label: string;
+        is_correct: boolean;
+      }[];
+    }[];
+  };
+}
```

Comment:
This should come from the drizzle type.

Look at how I set the type of json columns in the schema.ts file

Reply:
alright